plugins/hypervisors/kvm/pom.xml

```diff
@@ -45,7 +38,7 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.libvirt</groupId>
+      <groupId>com.github.K0zka</groupId>
```

Comment:
Hi Rohit,

I updated the group ID to that because I can publish with that group ID only and not with org.libvirt, unfortunately the libvirt maintainers do not seem to be interested in pushing their artifacts to central repo.
I agree that the groupId does not look good, but I haven't found another way. I will try to keep an up to date version in central repo untl libvirt maintainers accept the changes I have done on the pom.xml and publish to maven central, but this may take quite some time.